app/templates/bestuurseenheid/zitting/agenda/index.hbs

```diff
@@ -1,22 +1,31 @@
 <div class="au-o-layout">
   <a href="http://data.lblod.info/id/document-types/agenda" property="dct:type" class="au-u-hidden"></a>
-
-  <div class="au-o-region-large" typeof="besluit:Zitting" resource={{this.model.uri}}>
+  <div class="au-o-region-large" typeof="besluit:Zitting" resource={{this.meeting.uri}}>
     <AuHeading @skin="2" class="au-u-margin-bottom">
-      <span resource={{this.model.agendas.firstObject.uri}} typeof="foaf:Document" property="besluit:heeftAgenda">Agenda</span>
+      <span resource={{this.agenda.uri}} typeof="foaf:Document" property="besluit:heeftAgenda">Agenda</span>
       voor
-      <span resource={{this.model.bestuursorgaan.uri}} typeof="besluit:Bestuursorgaan" property="besluit:isGehoudenDoor">
-        <span resource={{this.model.bestuursorgaan.isTijdsspecialisatieVan.uri}} typeof="besluit:Bestuursorgaan" property="mandaat:isTijdspecialisatieVan">
-          <span property="skos:prefLabel">{{this.model.bestuursorgaan.isTijdsspecialisatieVan.naam}}</span>
+      <span resource={{this.meeting.bestuursorgaan.uri}} typeof="besluit:Bestuursorgaan" property="besluit:isGehoudenDoor">
+        <span resource={{this.meeting.bestuursorgaan.isTijdsspecialisatieVan.uri}} typeof="besluit:Bestuursorgaan" property="mandaat:isTijdspecialisatieVan">
+          <span property="skos:prefLabel">{{this.meeting.bestuursorgaan.isTijdsspecialisatieVan.naam}}</span>
         </span>
       </span>
       , zitting van
-      <span property="besluit:geplandeStart" content={{iso-date this.model.geplandeStart}} datatype="xsd:dateTime">{{moment-format this.model.geplandeStart 'D MMMM YYYY, HH:mm'}}</span>
+      <span property="besluit:geplandeStart" content={{iso-date this.meeting.geplandeStart}} datatype="xsd:dateTime">{{moment-format this.meeting.geplandeStart 'D MMMM YYYY, HH:mm'}}</span>
     </AuHeading>
-
+    <p class="au-c-heading au-c-heading--5">
+      Datum publicatie:
+      {{#if this.publication.createdOn}}
+        <span property="eli:date_publication" content={{moment-format this.publication.createdon 'YYYY-MM-DD'}} datatype="xsd:date">
```

Comment:
I think this is a typo? `createdOn` vs `createdon`?

Reply:
yes, thanks for catching this!